packages/run-protocol/test/supports.js

```diff
@@ -166,3 +164,26 @@ export const produceInstallations = (space, installations) => {
     space.installation.produce[key].resolve(installation);
   }
 };
+
+/**
+ * @param {object} kit
+ * @param {Brand<'nat'>} kit.brand
+ * @param {Issuer<'nat'>} [kit.issuer]
+ * @param {Mint<'nat'>} [kit.mint]
+ */
+export const withAmountUtils = kit => {
+  return {
+    ...kit,
```

Comment:
> If we normally passed this around in production code with the mint included that would be scary.

Agreed. That's why I put it under `test/` and not `src/`.

For production code I was thinking something similar that took just `brand`, but I think that would need to be part of a more holistic look at how we organize code for amounts, ratios, etc.

Reply:
If we agree not to include the mint, we can use the same pattern in production and test.

Reply:
True. Do you want to include that in the scope of this PR? My preference is to land this as is and tackle the production another day.

Reply:
I would not make widespread changes here, but it would be apropos to change the helpers now to match our target, n'est ce pas?

Reply:
The thing is, in tests there are legitimate needs for the `mint`. e.g.,
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/vaultFactory/test-liquidator.js#L277
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/vaultFactory/test-liquidator.js#L389
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/vaultFactory/test-liquidator.js#L479
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/vaultFactory/test-vaultFactory.js#L301
https://github.com/Agoric/agoric-sdk/blob/262097f7b82e2fba1a56b6f191b8ddbd4296ddaf/packages/run-protocol/test/vaultFactory/test-vaultFactory.js#L301

It makes sense since tests often have the full issuer kit. If in production we add some utility similar to `withAmountUtils`, I don't think it will replace this one. I expect this one to use it and still return `mint` when it's defined.

Reply:
I agree that the mint is needed in several places in tests. It's also needed in a few places in production. I just think the helper should consistently not include the mint. In order to make that work in production we'll need a mechanism for passing around the helper without the mint and pass around the mint separately, and only when it is necessary. I think the same approach would work in tests as well, and the consistency would be a benefit for us.

But I'm willing to put off that discussion until we decide to add such a helper to the production side.